### ISSUE TITLE
As a watcher of anime through using cli I would like to pass a --volume={1-150} to start mpv at that volume level when anime is loaded

### ISSUE THREAD

**Issue by 4cecoder** [pystardust/ani-cli#388]

**Comment by 4cecoder** [comment]
Example Use:
`ani-cli boruto --volume=50`

Choose Episode:
1-232
Mpv opening Playing... 
(at selected volume)

**Comment by 71zenith** [comment]
If this feature were to be implemented then I would recommend having an option to pass any argument to the player.

**Comment by ura43** [comment]
> If this feature were to be implemented then I would recommend having an option to pass any argument to the player.

Yes. A flag to pass options to the video player, doesn't have to be mpv, seems like the way to go

**Comment by ura43** [comment]
Yeah, that title is way to long

**Comment by ura43** [comment]
> As a watcher of anime through cli:

that is not something that should *ever* be in the title
